english/eatmarshmallow/eatmarshmallow.md

```diff
@@ -0,0 +1 @@
+You charge the Marshmallows with your mouth wide open, ready to eat them.  They decide to simply jump you and pin you to the ground, suffocating you.  The End.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.